pkg/platform/packet/template.go

```diff
@@ -125,8 +135,27 @@ module "worker-{{ $pool.Name }}" {
 
   ssh_keys  = {{$.SSHPublicKeys}}
 
+  {{- if $pool.CLCSnippets}}
+  clc_snippets = [
+  {{- range $pool.CLCSnippets}}
+    <<EOF
+{{.}}
```

Comment:
Can we use a name?

Reply:
could you please elaborate @rata 

A name ?

Reply:
ah, I get it.

Reply:
I think having a name is not going to improve readability as we are not doing much with it after wards. Still if others are in favor of using a name thats fine by me.

Reply:
I guess we can :)

Reply:
done.

Reply:
Thanks!